apps/desktop/ui/styles.css

```
@@ -33,3 +33,8 @@ body,
     width: 0%;
   }
 }
+
+@keyframes attachToastFadeIn {
```

Stylelint (17.9.0)

[error] Expected keyframe name "attachToastFadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)